files/system/usr/bin/flatpak-set-perms.sh

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env bash
```

Comment:
as a general rule for working in FOSS projects, it's good to get buy-in from the project before working on features so you don't invest time on changes that won't be accepted